File: Logic & Design/Visual Studio Code/Funcitons/Java/mathQuizFunction.java
/*
 * Programmer:      Corey Jenkins
 * Date:            August 15, 2023
 * Filename:        mathQuizFunction.java
 * Purpose:         A math quiz program. This program will display two random numbers. A function is then made for the user to 
 *                  input the answer. An if/else statement is created and will return the answer if it is correct or incorrect.
 */

import java.util.Scanner;
import java.util.Random;

public class mathQuizFunction {
    public static void main(String[] args) {
        Random randomMathNumbers = new Random();

        int firstNumber, secondNumber, answer;

        firstNumber = randomMathNumbers.nextInt(300) + 1;
        secondNumber = randomMathNumbers.nextInt(300) + 1;
        answer = quiz(firstNumber, secondNumber);
        System.out.println(answer);
    }

    public static int quiz(int num1, int num2) {
        Scanner input = new Scanner(System.in);
        int result, inputAnswer;
        System.out.println(num1 + "+" + num2);
        System.out.println("Enter your answer:");
        inputAnswer = input.nextInt();

        result = num1 + num2;

        if (inputAnswer == result) {
            System.out.println("This is correct.");
            return result;
        } else {
            System.out.println("Incorrect.");
            return result;
        }
    }
}

File: Logic & Design/Visual Studio Code/Funcitons/Python/math_quiz_function.py
'''
Programmer:     Corey Jenkins
Date:           August 15, 2023
Filename:       math_quiz_function.py
Purpose:        To create a program in which it shows a math problem at random. Then it will return a function if the answer given was correct or incorrect.
'''

import random
def main():
    
    first_number = random.randint(1, 300)
    second_number = random.randint(1, 300)
    answer_function = sum(first_number, second_number)
    print(answer_function)
    
def sum(first_number, second_number):
    result = first_number + second_number
    print(first_number, "+", second_number)
    answer_input = int(input("Enter your answer: "))
    if result == answer_input:
        print("Correct!")
        return result
    else:
        print('Incorrect!')
        return result
main()
    
    
    